ui/packages/catalog-ui-search/src/main/webapp/component/layer-item/layer-item.less

```diff
@@ -0,0 +1,131 @@
+@{customElementNamespace}layer-item {
+    display: block;
+    white-space: nowrap;
+    width: 100%;
+    overflow: hidden;
+    position: relative;
+
+    .layer-properties,
+    .layer-rearrange {
+        display: inline-block;
+        vertical-align: middle;
+    }
+
+    .layer-properties {
+        padding: @mediumSpacing;
+        margin-left: @minimumButtonSize;
+        width: ~'calc(100% - @{minimumButtonSize})';
+    }
+
+    .layer-rearrange-down,
+    .layer-rearrange-up {
+        z-index: 1;
+        position: absolute;
+        left: 0px;
+        opacity: 0;
+        width: @minimumButtonSize;
+        height: 0.5*@minimumButtonSize;
+        line-height: 0.5*@minimumButtonSize;
+        > span {
+            position: absolute;
+            left: 50%;
+            top: 50%;
+            transform: translateX(-50%) translateY(-50%);
+        }
+    }
+
+    .layer-rearrange-down {
+        top: ~'calc(100% - .5*@{minimumButtonSize})';
+    }
+
+    .layer-rearrange-up {
+        top: 0px;
+    }
+
+    .layer-rearrange-up:hover,
+    .layer-rearrange-up:focus,
+    .layer-rearrange-down:hover,
+    .layer-rearrange-down:focus {
+        opacity: 1;
+    }
+
+    .layer-rearrange {
+        .grab-cursor();
+        position: absolute;
+        left: 0px;
+        top: 0px;
+        height: 100%;
+        width: @minimumButtonSize;
+    }
+
+    .layer-show {
+        height: @minimumButtonSize;
+        width: @minimumButtonSize;
+        position: relative;
+        > span {
+            position: absolute;
+            left: 50%;
+            top: 50%;
+            transform: translateX(-50%) translateY(-50%);
+        }
+    }
+
+    .layer-alpha,
+    .layer-show {
+        display: inline-block !important;
+        vertical-align: middle;
+    }
+
+    .layer-alpha {
+        width: ~'calc(100% - @{minimumButtonSize})' !important;
+    }
+
+    .layer-name {
+        margin-bottom: @minimumSpacing;
+        overflow: hidden;
+        text-overflow: ellipsis;
```

Comment:
❓ Is there a hovertext/tooltip for overflowing text?

Reply:
Yup!  Check out the layer-item template.